packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -152,6 +152,17 @@ const NuovaDelega = () => {
     dispatch(resetNewDelegation());
   }, []);
 
+  const deleteInput = (
+    funField: (field: string, setValue: any, validation: boolean | undefined) => void,
+    funTouched: (field: string, setValue: boolean, validation: boolean) => void
+  ): MouseEventHandler<HTMLLabelElement> => {
+    funField('nome', initialValues.nome, false);
+    funField('cognome', initialValues.cognome, false);
+    funTouched('nome', false, false);
+    funTouched('cognome', false, false);
```

Comment:
Enable validation for the last function call

Reply:
do it

Reply:
done correctly